### PR TITLE
Refactor sgl-kernel build

### PR DESCRIPTION
**Pull request by ispobock**
## Motivation

Involve all ops into one target. Reduce redundant build code.

### CODE REVIEW

File: sgl-kernel/src/sgl-kernel/ops/__init__.py [hunk 1]
```diff
@@ -1,4 +1,24 @@
-from .moe_align_block_size import moe_align_block_size as _moe_align_block_size
+from ._kernels import all_reduce as _all_reduce
```
Comment (merrymercy) [inline sgl-kernel/src/sgl-kernel/ops/__init__.py]: can we make these absolute imports?

Comment (ispobock) [reply sgl-kernel/src/sgl-kernel/ops/__init__.py]: Yes, absolute imports also works.

**Comment by zhyncs** [comment]
I’ll help review today

**Review by zhyncs — COMMENTED** [review]
Please also update `CMakeLists.txt`

**Comment by zhyncs** [comment]
close https://github.com/sgl-project/sglang/issues/2602

File: sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu [hunk 2]
```diff
@@ -3,8 +3,8 @@
 #include <ATen/ATen.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <c10/cuda/CUDAGuard.h>
-#include <torch/all.h>
-#include <torch/extension.h>
+
```
Comment (zhyncs) [inline sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu:6]:
```suggestion

```

File: sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu [hunk 3]
```diff
@@ -3,8 +3,8 @@
...
+#include "utils.hpp"
```
Comment (zhyncs) [inline sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu]:
```suggestion
```

File: sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu [hunk 4]
```diff
@@ -3,8 +3,8 @@
...
 
 #include <THC/THCAtomics.cuh>
```
Comment (zhyncs) [inline sgl-kernel/src/sgl-kernel/csrc/moe_align_kernel.cu:7]:
```suggestion
#include <THC/THCAtomics.cuh>

#include "utils.hpp"
```

**Review by zhyncs — APPROVED** [review]

**Pull request merged**